src/amp/abTests.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    ExperimentFullConfig
+} from "@root/src/amp/components/Experiment";
+
+// Variant proportions must be >0, so we set extremely low to simulate a 0% test
+export const abTestsFullConfig: ExperimentFullConfig = {
```

Comment:
We'd probably better off avoiding introducing a terminology (ABTest) that's more restrictive than the AMP terminology (experiment)

Reply:
Improved naming in recent refactor